apps/meteor/client/components/Omnichannel/modals/ForwardChatModal.tsx

```diff
@@ -43,7 +44,26 @@ const ForwardChatModal = ({
 	const hasDepartments = useMemo(() => departments && departments.length > 0, [departments]);
 
 	const _id = { $ne: room.servedBy?._id };
-	const conditions = { _id, status: { $ne: 'offline' }, statusLivechat: 'available' };
+	const conditions = {
+		_id,
+		...(!idleAgentsAllowedForForwarding && {
+			$or: [
+				{
+					status: {
+						$exists: true,
+						$ne: 'offline',
+					},
+					roles: {
+						$ne: 'bot',
+					},
+				},
+				{
+					roles: 'bot',
+				},
+			],
+		}),
+		statusLivechat: 'available',
```

Comment:
> here we are considering statusLivechat only here, but probably there are many places where a offline, but active agent should appears

@ggazzo Sorry, I didn't get this part :see_no_evil: 

Also, any chance we can take this task separately to remove the dependency of `UserAutoComplete` component within the forwarding chat modal? Because the release cut is quite close & I don't think I'd be able to replace this entire behaviour in time :sweat_smile: (We don't even have any endpoints already present for this right now :grimacing: )